shared/src/main/scala/pl/metastack/metatime/Scheduler.scala

```diff
@@ -20,6 +20,27 @@ trait Scheduler {
     })
 
   def currentTimeMillis(): Long = System.currentTimeMillis()
+
+  def at(time: Time)(f: => Unit): Cancelable =
+    at(time.fromNow())(f)
+  def at(dateTime: DateTime)(f: => Unit): Cancelable =
+    at(dateTime.fromNow())(f)
+  def at(time: Offset[_])(f: => Unit): Cancelable =
+    scheduleOnce(Math.abs(time.component.asInstanceOf[Component].milliseconds()).millis)(f)
```

Comment:
I tried to change line 28 to def at(time: Offset[Component)(f: => unit) But I get error on line 25 and 27 saying "Type Mismatch: expected Time, actual Offset[_]" and "Type Mismatch: expected DateTime, Actual Offset[_]" respectively.